pandas/tests/io/test_fsspec.py

```diff
@@ -0,0 +1,70 @@
+import gc
+
+import numpy as np
+import pytest
+
+from pandas import DataFrame, date_range, read_csv
+import pandas._testing as tm
+from pandas.util import _test_decorators as td
+
+from pandas.io.common import is_fsspec_url
+
+df1 = DataFrame(
+    {
+        "int": [1, 3],
+        "float": [2.0, np.nan],
+        "str": ["t", "s"],
+        "dt": date_range("2018-06-18", periods=2),
+    }
+)
+text = df1.to_csv(index=False).encode()
+
+
+@pytest.fixture
+@td.skip_if_installed("fsspec")
```

Comment:
Why is this skipped if fsspec is installed?

Reply:
Ah, should be the opposite - but why did the tests run for me??